store/index.ts

```diff
@@ -2,15 +2,13 @@ interface StateI {
   isStaff: boolean
   isSbcStaff: boolean
   roles: string[]
-  accountId: number
 }
 
 export function state (): StateI {
   return {
     isStaff: null,
     isSbcStaff: null,
     roles: [],
-    accountId: null,
```

Comment:
why remove accountId?

Reply:
It was only needed in dashboard.vue so I just have it as a local variable there.

Reply:
Cool